backend/core/test/authz/authz.e2e-spec.ts

```diff
@@ -188,14 +188,9 @@ describe("Authz", () => {
         .set(...setAuthorization(userAccessToken))
         .expect(403)
     })
-    it(`should allow normal/anonymous user to POST listings`, async () => {
+    it(`should not allow anonymous user to POST listings`, async () => {
       // anonymous
       await supertest(app.getHttpServer()).post(listingsEndpoint).expect(403)
-      // logged in normal user
-      await supertest(app.getHttpServer())
-        .post(listingsEndpoint)
-        .set(...setAuthorization(userAccessToken))
-        .expect(403)
```

Comment:
You have to remove this test because of the changes to the auth you made for this script to work? Could you add a note in the issue for locking down the access control to add this test back?

Reply:
I mentioned this in #57